src/Panel.js

```diff
@@ -92,23 +91,32 @@ class Panel extends React.Component {
   }
 
   getExpanded() {
-    const { eventKey } = this.props;
-    const { activeKey } = this.context.$bs_panelGroup || {};
+    const panelGroup = this.context.$bs_panelGroup;
 
-    return this.props.expanded != null || activeKey === undefined
-      ? this.props.expanded
-      : activeKey === eventKey;
+    if (panelGroup && has.call(panelGroup, 'activeKey')) {
```

Comment:
This is annoying but `activeKey` starts out as `undefined` if the panel group has no default value... not sure what we can do there

Reply:
I had got to the same point of checking hasOwnProperty() myself.